packages/icons/icons.json

```diff
@@ -3890,5 +3890,19 @@
         "tags": "number, float, double, decimal",
         "group": "miscellaneous",
         "codepoint": 62252
+    },
+    {
+        "displayName": "Floating point array",
+        "iconName": "array-floating-point",
+        "tags": "number, float, double, decimal, array",
+        "group": "miscellaneous",
+        "codepoint": 62253
+    },
+    {
+        "displayName": "Cube: edit",
+        "iconName": "cube-edit",
```

Comment:
compilation is failing, it looks like you accidentally added this extra icon definition here. please remove it.

Reply:
@adidahiya removed. Please review again, thanks 🙏